### PR TITLE
Implement drop down

### PR DESCRIPTION
**Pull request by vitsalis**

### CODE REVIEW

**Comment by vitsalis** [comment]
Depends #260

**Comment by dionyziz** [comment]
@pkakelas Please review.

File: static/script/game/view.js [hunk 1]
```diff
@@ -3,6 +3,7 @@ var GameView = {
     ARROW_WIDTH: 30,
     roundCount: 0,
     maxHp: 0,
+    pixelMultiplier: 20,
```
Comment (dionyziz) [inline static/script/game/view.js]: nit: `PIXEL_MULTIPLIER`

**Comment by dionyziz** [comment]
LGTM

File: views/header.php [hunk 2]
```diff
@@ -39,30 +40,47 @@
...
+                            $user = $_SESSION[ 'user' ];
+                            ?><li><a href="user/view?username=<?php
+                                echo $user->username;
+                            ?>"><img src="<?php
+                                echo $user->image->target_path;
+                            ?>" width="30" height="30" /><?php
+                                echo $user->username;
+                            ?></a></li>
+                            <li class="dropdown">
+                                <a href="#" class="dropdown-toggle" data-toggle="dropdown">
```
Comment (pkakelas) [inline views/header.php:60]: Good job for using bootstrap to make the dropdown

File: static/script/game/view.js [hunk 3]
```diff
@@ -3,7 +3,7 @@ var GameView = {
     ARROW_WIDTH: 30,
     roundCount: 0,
     maxHp: 0,
-    pixelMultiplier: 20,
+    PIXEL_MULTIPLIER: 20,
```
Comment (pkakelas) [inline static/script/game/view.js:6]: Why do you use capital letters?

File: views/header.php [hunk 4]
```diff
@@ -39,30 +40,47 @@
...
+                                    <span class="settings-icon glyphicon glyphicon-cog"></span>
+                                </a>
+                                <ul class="dropdown-menu">
+                                    <li role="presentation">
+                                        <a href="user/update">Settings</a>
+                                    </li>
+                                    <li role="presentation">
+                                        <a href="#" id="logout">Sign out</a>
+                                    </li>
+                                    <?php
```
Comment (pkakelas) [inline views/header.php]: If the form is inside a ul why don't you put it inside a li?

File: views/header.php [hunk 5]
```diff
@@ -39,30 +40,47 @@
...
-               ?></ul>
+                <ul class="nav navbar-nav navbar-right">
+                    <?php
+                        if ( isset( $_SESSION[ 'user' ] ) ) {
+                            $user = $_SESSION[ 'user' ];
+                            ?><li><a href="user/view?username=<?php
+                                echo $user->username;
+                            ?>"><img src="<?php
+                                echo $user->image->target_path;
+                            ?>" width="30" height="30" /><?php
```
Comment (pkakelas) [inline views/header.php]: Maybe put these in header.css?

File: views/header.php (hunk 4 above)
Comment (dionyziz) [inline views/header.php]: Yes, the form has to be moved within the li above.

File: static/script/game/view.js (hunk 3 above)
Comment (dionyziz) [inline static/script/game/view.js:6]: It's a naming convention for constants.

**Pull request merged**